README.md

```diff
@@ -138,22 +138,48 @@ pnpm install
 
 3. **Run the extension**:
 
-Press `F5` (or **Run** → **Start Debugging**) in VSCode to open a new window with Roo Code running.
+There are several ways to run the Roo Code extension:
 
-Changes to the webview will appear immediately. Changes to the core extension will require a restart of the extension host.
+### Development Mode (F5)
 
-Alternatively you can build a .vsix and install it directly in VSCode:
+For active development, use VSCode's built-in debugging:
 
-```sh
-pnpm vsix
-```
+Press `F5` (or go to **Run** → **Start Debugging**) in VSCode. This will open a new VSCode window with the Roo Code extension running.
+
+- Changes to the webview will appear immediately.
+- Changes to the core extension will require a restart of the extension host (close and reopen the new VSCode window).
```

Comment:
I don't think this is true anymore. We added hot reload for the extension as well.